README-aarch64.md

```diff
@@ -0,0 +1,208 @@
+# OpenVPN for Docker
```

Comment:
Instead of adding a new README, we should put an `aarch64.md` in the `docs/` folder.